kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/KroxyliciousST.java

```diff
@@ -218,5 +210,8 @@ void setupBefore() {
         resourceManager.createResourceWithWait(
                 KafkaNodePoolTemplates.kafkaBasedNodePoolWithDualRole(BROKER_NODE_NAME, kafka, 3).build(),
                 kafka);
+
+        kroxyliciousOperator = new KroxyliciousOperator(Constants.KROXYLICIOUS_OPERATOR_NAMESPACE);
```

Comment:
I think this should move to its own method with a `@BeforeAll` annotation as this will be skipped if there already some Kafka pods (see line 201) but that does not imply there is already an operator deployed as that is removed as part of `cleanUp`.